engines/bastion_katello/app/assets/javascripts/bastion_katello/subscriptions/subscriptions.controller.js

```diff
@@ -39,11 +39,6 @@ angular.module('Bastion.subscriptions').controller('SubscriptionsController',
             $scope.transitionTo('subscriptions.index');
         };
 
-        $scope.groupedSubscriptions = {};
-        $scope.$watch('table.rows', function (rows) {
-            $scope.groupedSubscriptions = SubscriptionsHelper.groupByProductName(rows);
-        });
-
```

Comment:
@thomasmckay do you have any thoughts on removing the grouping of subscriptions?